doc/lspsaga.nvim.txt

```diff
@@ -564,6 +564,7 @@ Default options:
         auto_preview = true,
         auto_refresh = true,
         auto_close = true,
+        auto_resize = true,
```

Comment:
there is no need. it will auto generate by action.

Reply:
Ok, I will remove it then